client/psf.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -e
```

Comment:
I once learned not to use `-e` in the shebang, but explicitly call `set -e`, so that whoever thinks it's smart to call `bash psf.sh` gets the same treatment.